public/pages/MonitorDetails/components/MonitorOverview/MonitorOverview.js

```diff
@@ -25,14 +30,70 @@ const MonitorOverview = ({
     detector,
     detectorId
   );
+
+  let relatedMonitorsStat = null;
+  let relatedMonitorsData = null;
+
+  if (monitor.associated_workflows) {
+    const links = monitor.associated_workflows.map(({ id, name }) => ({
+      name,
+      href: `${PLUGIN_NAME}#/monitors/${id}?type=${'workflow'}`,
+    }));
+
+    relatedMonitorsData = {
+      header: 'Associations with composite monitors',
+      tableHeader: 'Composite monitors',
+      links,
+    };
+  } else if (delegateMonitors.length) {
+    const links = delegateMonitors.map(({ id, name }) => ({
+      name,
+      href: `${PLUGIN_NAME}#/monitors/${id}?type=${'monitor'}`,
```

Comment:
Clarification question: Is `?type=${'monitor'}` necessary? Could it be simplified to `?type=monitor`?